src/Essentials/src/Battery/Battery.android.cs

```diff
@@ -63,7 +63,7 @@ void StopBatteryListeners()
 			{
 				System.Diagnostics.Debug.WriteLine("Battery receiver already unregistered. Disposing of it.");
 			}
-			batteryReceiver.Dispose();
+			batteryReceiver?.Dispose();
```

Comment:
Is there one more issue? It looks like this would call `Application.Context.UnregisterReceiver(null)` above if `batteryReceiver` was `null`.

Can we enable nullable-reference types in this file as well? We should get a build error (with TreatWarningsAsErrors) if this is wrong.

Reply:
Ok, enabled nullable on the Android `BatteryImplementation`.